integration-tests/src/tests/client/benchmarks.rs

```diff
@@ -6,7 +6,8 @@
 
 use near_chain_configs::Genesis;
 use near_client::test_utils::{create_chunk_on_height, TestEnv};
-use near_client::{ProcessTxResponse, ProduceChunkResult};
+use near_client::ProcessTxResponse;
+use near_client::ProduceChunkResult;
```

Comment:
mini nit: Can revert to `use near_client::{ProcessTxResponse, ProduceChunkResult};`? Fine to leave as is.